pump/storage/storage_test.go

```diff
@@ -281,10 +276,15 @@ func populateBinlog(b Log, append *Append, prewriteValueSize int, binlogNum int3
 				binlog.Tp = pb.BinlogType_Prewrite
 				startTS := getTS()
 				binlog.StartTs = startTS
-				binlog.PrewriteValue = prewriteValue
+				if fixedValueSize {
+					binlog.PrewriteValue = prewriteValue
+				} else {
+					size := 1 + rand.Intn(len(prewriteValue)-1)
```

Comment:
you want `1 + rand.Intn(len(prewriteValue)` ?

Reply:
It should be 1 less than the size because we want it to be `1 + [0,n-2] = [1, n-1]`.